js/localization/messages/es.json

```diff
@@ -195,38 +195,38 @@
         "dxScheduler-confirmRecurrenceEditOccurrence": "Editar cita",
         "dxScheduler-confirmRecurrenceDeleteOccurrence": "Eliminar cita",
         "dxScheduler-noTimezoneTitle": "Sin zona horaria",
-        "dxScheduler-moreAppointments": "",
+        "dxScheduler-moreAppointments": "{0} +",
```

Comment:
Could we use "{0} más" here?

Reply:
@BingoRUS Yes, that is another option for this translation. Let's use your suggestion, but both are valid. I will change it here.